packages/flutter_secure_storage/example/lib/main.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: use_build_context_synchronously
```

Comment:
Instead, consider using `if (!mounted) return` for overall consistency.

Reply:
Applied it. Thank you.

Reply:
Oh. The upstream code has just been updated. Please check.

Reply:
Updated!